### DISCUSSION TITLE
Medical imaging implementation

### DISCUSSION THREAD

**Discussion by Samy-mri** [open-edge-platform/anomalib#1126] [Feature Requests]
Hello everyone,

I've recently started using Anomalib to detect morphological abnormalities in T1-weighted MRI images, in patients with movement disorders.

To implement this I added a few lines of code to handle NIFTI reading (src/data/utils/image.py). 
and to handle gray-scale images I am using 3 neighbouring slices to accomodate for the pretrained (RGB) models, which suited my application. Another trick would be to unsqueeze in the channel dimension.

I also added some code to handle non-int8 data (/src/post_processing/post_process.py and visualizer.py).

I am wondering if there is interest to implement this properly, which would also mean accomodating for single-channel uint16 data.

Happy to share what I have already.
Best,
Samy

**Comment by yarinbar** [comment]
Do you have any update or tips? I am trying to use it too on medical data :)
